__tests__/business-text/business-text.mutation.test.js

```diff
@@ -60,7 +63,7 @@ describe('Business text mutations test', () => {
   });
 
   test('#2 adding a new page with existing code should return error', async () => {
```

Comment:
Please, remove `#2` from the description string

Reply:
fixed